spec/index.html

```diff
@@ -350,11 +350,11 @@ <h3>Triple Terms and Reification</h3>
     </figure>
 
     <p>
-      In this example, the [=proposition=] denoted by the [=triple term=] (<code>:Alice</code>'s name being <code>"Alice"</code>) is not claimed to be true.
+      In this example, the [=proposition=] denoted by the [=triple term=] (<code>:Alice</code>'s family name being <code>"Liddell"</code>) is not claimed to be true.
```

Comment:
When I read this, I first thought that the parenthesis is the triple term that this sentence talks about and so I wondered why there are no commas between subject, predicate, and object; why the predicate is not an IRI; etc. Eventually, I realized that this parenthesis is actually meant as a clarification for the proposition that the sentence talks about. To avoid others running into the same confusion, I suggest to change the parenthesis as follows.
```suggestion
      In this example, the [=proposition=] denoted by the [=triple term=] (i.e., the proposition that <code>:Alice</code>'s family name is <code>"Liddell"</code>) is not claimed to be true.
```